ijServer/src/main/kotlin/org/jetbrains/research/tasktracker/Utils.kt

```diff
@@ -21,3 +33,34 @@ fun initializeIds() {
     currentResearchIdAtomic = AtomicInteger(ResearchTable.maxId() + 1)
     currentSurveyIdAtomic = AtomicInteger(0)
 }
```

Comment:
Why did you do it? You can increment ID automatically in db, see an example [here](https://www.baeldung.com/kotlin/exposed-persistence#1-columns). Please, refactor all models